src/main/java/com/google/songgame/servlets/RoomServlet.java

```diff
@@ -36,22 +37,44 @@ public void init() {
     gson = new Gson();
     datastore = DatastoreServiceFactory.getDatastoreService();
   }
-  
+
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Map<String, String> roomProperties = readJSONFromRequest(request);
 
-    // Save player roomId to datastore.
+    List<String> userIdList = new ArrayList<String>();
+    userIdList.add(roomProperties.get("userId"));
+
+    // Save roomId to datastore.
     Entity roomEntity = new Entity("Room");
     roomEntity.setProperty("roomId", roomProperties.get("roomId"));
-    
+    roomEntity.setProperty("userId", userIdList);
```

Comment:
Might want to change the name of this property to userIdList instead of keeping it as userId